tests/requirements/oracle.txt

```diff
@@ -1 +1 @@
-cx_oracle >= 7.0
+oracledb >= 1.2.0
```

Comment:
We can wait a bit, the feature freeze for Django 5.0 is on September, 18th.

Reply:
I would set 1.3.2 as minimum supported.  This will give maximum testing time before Django 5.0.  The changes in 1.4 are around areas that won't impact Django users.

Reply:
I'd argue about the actual impact, but suspect I won't win.

Reply:
> I'd argue about the actual impact, but suspect I won't win.

Agreed, the actual impact is not huge. I will try to merge this in the next few days.

Reply:
@cjbj Since I have your (Oracle) attention, have you ever considered [becoming](https://www.djangoproject.com/foundation/corporate-membership/join/) a DSF [corporate member](https://www.djangoproject.com/foundation/corporate-membership/)? We've been supporting the Oracle backend on our own for 18 years.

Reply:
I'm happy to take that to management.  There might be some follow up questions.  Would you care to chat to me first?

Reply:
> Would you care to chat to me first?

Sure, it's always good to talk, you can use my email, it's [public](https://github.com/felixxm).

Reply:
@felixxm I'll set up a call for next week.

Reply:
(Un)fortunately, I'm on vacation next week. We can try after August, 21st or you can contact directly with the DSF board. I'm sure they would love to chat (\cc @django/dsf-board).

Reply:
@felixxm next week is busy for me too, so I can wait until you return.

Reply:
> @felixxm next week is busy for me too, so I can wait until you return.

I'm available for a call if you still want to catch up (daily from 6 am UTC to 12 o'clock UTC). We can figure out another time-slot if it doesn't work for you.

Reply:
@felixxm I've sent a calendar invitation to meet tomorrow.

Reply:
@sarahboyce I only used this conversion as a reference for the version bump.